elements/lisk-p2p/src/disconnect_status_codes.ts

```diff
@@ -35,8 +35,10 @@ export const INCOMPATIBLE_PEER_CODE = 4104;
 export const INCOMPATIBLE_PEER_UNKNOWN_REASON =
 	'Peer is incompatible with the node for unknown reasons';
 
-// First case to follow HTTP status codes
 export const FORBIDDEN_CONNECTION = 4403;
 export const FORBIDDEN_CONNECTION_REASON = 'Peer is not allowed to connect';
 
+export const DUPLICATE_CONNECTION = 4404;
```

Comment:
Errors look too similar. https://github.com/LiskHQ/lisk-sdk/blob/decb81e6489730810a18f7968db424860ebf7c2d/elements/lisk-p2p/src/disconnect_status_codes.ts#L38

Reply:
The first/original connection can be outbound so the suggested name would create confusion because it sounds like there are two duplicate inbound connections which is not the case.

Reply:
Sorry, it wasn't a suggestion. I've just showed a similar variable that we are introducing in another PR https://github.com/LiskHQ/lisk-sdk/pull/4169. Shall we change this variable's name or the other?

Reply:
Ah, I think it may be the same error. We should change it in the other PR so that they match.